### PR TITLE
Create ui-c8y-1021-0-0-acivating-user-switch-reworked.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafkk

Ticket: [DM-3875](https://cumulocity.atlassian.net/browse/DM-3875)
Original PR: [7238](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7238)

[DM-3875]: https://cumulocity.atlassian.net/browse/DM-3875?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Acivating user switch reworked.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]: The Claude AI suggested to use the following title:
```suggestion
title: Rework user switch activation
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3875
+version: 1021.0.0
+---
+Acivating user switch reworked.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]: The Claude AI suggested to use the following body:
```suggestion
To simplify the user experience when switching between users, the user switch activation has been reworked. Previously, the user had to manually activate the user switch in a separate step after selecting the user to switch to. Now, the user switch is automatically activated when selecting the user from the user switch dropdown menu. This change simplifies the user switching process and makes it more intuitive for users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Rework user switch activation
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]:
```suggestion
title: Activating user in the Device credentials tab fixed
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3875
+version: 1021.0.0
+---
+To simplify the user experience when switching between users, the user switch activation has been reworked. Previously, the user had to manually activate the user switch in a separate step after selecting the user to switch to. Now, the user switch is automatically activated when selecting the user from the user switch dropdown menu. This change simplifies the user switching process and makes it more intuitive for users.
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]:
```suggestion
Fixed issue in device credentials functionality allowing reliable user activation and deactivation.
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Activating user in the Device credentials tab fixed
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]:
```suggestion
title: Activating user in the Device credentials tab
```

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3875
+version: 1021.0.0
+---
+Fixed issue in device credentials functionality allowing reliable user activation and deactivation.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]:
```suggestion
In the **Device credentials** page, user activation did not work properly. This issue has been fixed an activating or deactivating users now works reliably.
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Activating user in the Device credentials tab
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md]:
```suggestion
title: Activating or deactivating users in the Device credentials page works reliably
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**